docs(cron): Add note about luxon timezones (#11451)
getsentry/sentry-docs: includes/javascript-crons-upsert.mdx

The `tz` where your job is running. This is usually your server's timezone, (such as `America/Los_Angeles`). See [list of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). Optional.

> <Note>
> 
> While `node-cron` allows to pass luxon timezones, you have to convert them to an IANA timezone when using Sentry (e.g. using `DateTime.local().zoneName`)
> 
> </Note>

Reviewer: ```suggestion
<Note>

`node-cron` allows you to pass luxon timezones, but you have to convert them to an IANA timezone when using Sentry (for example, using `DateTime.local().zoneName`).

</Note>
```
